### PR TITLE
fix: Use HTTPs links instead of HTTP

### PR DESCRIPTION
**Pull request by Panquesito7**
## Changes proposed

- Use HTTPs links instead of HTTP.
  - Do let me know if all links work properly (except those like GitHub/Twitter). 😄

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/700"></a>

### CODE REVIEW

**Review by emmalearnscode — CHANGES_REQUESTED** [review]
Can you remove the package-lock file from the PR?

File: Contributing.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # 💥 How to Contribute
 
-[![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](http://makeapullrequest.com)
+[![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](https://makeapullrequest.com)
```
Comment (schmelto) [inline Contributing.md:3]:
```suggestion
[![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](https://github.com/EddieHubCommunity/LinkFree/pulls)

```

**Comment by Panquesito7** [comment]
> Can you remove the package-lock file from the PR?

By the way, I think that file should be updated in a PR or commit. 😅

**Review by schmelto — APPROVED** [review]

**Review by krishguptadev — COMMENTED** [review]
LGTM! But I believe this could be a `docs:` commit and didn't need a version bump

**Pull request merged**